clients/drcachesim/common/trace_entry.h

```diff
@@ -71,12 +71,22 @@ typedef enum {
     TRACE_TYPE_READ,  /**< A data load. */
     TRACE_TYPE_WRITE, /**< A data store. */
 
-    TRACE_TYPE_PREFETCH, /**< A general prefetch to the level 1 data cache. */
+    TRACE_TYPE_PREFETCH, /**< A general prefetch. */
```

Comment:
I updated the comment for TRACE_TYPE_PREFETCH too. IIUC, it was intended to be a 'default' prefetch trace type, for operations that don't fall under any other existing type.

Reply:
OK, sure.